ui/raidboss/data/triggers/o9s.js

```diff
@@ -172,7 +172,7 @@
     // Fire Path
     {
       id: 'O9S Fire Phase Tracking',
-      regex: / 14:3186:Chaos starts using Blaze /,
+      regex: / 14:3186:Chaos starts using Blaze/,
```

Comment:
This isn't a fix, just a cleanup, right? I think the space is still correct even if messy?

Reply:
The log line says ` 14:3186:Chaos starts using Blaze on Chaos.` so the white space is fine for now.
Maybe the ` on Chaos.` part will disappear some day?